docker-images/ziti-console/Dockerfile

```diff
@@ -0,0 +1,24 @@
+ARG NODE_VERSION=20
+FROM node:${NODE_VERSION}-bookworm-slim as builder
+
+ARG BASE_HREF=/zac/
+ARG DEPLOY_URL=${BASE_HREF}
+
+WORKDIR /usr/src/app
+
+RUN npm install -g @angular/cli@16.0.0-next.0
+
+COPY package*.json ./
+RUN npm install --omit=optional
+
+COPY . .
+RUN ng build ziti-console-lib
+RUN ng build ziti-console --base-href ${BASE_HREF} --deploy-url ${DEPLOY_URL}
+
+ARG NODE_VERSION=${NODE_VERSION}
+FROM node:${NODE_VERSION}-bookworm-slim
+WORKDIR /usr/src/app
+
+COPY --from=builder /usr/src/app/dist/app-ziti-console ./dist/app-ziti-console
+
+CMD echo "deployment guide https://openziti.io/docs/guides/deployments/docker/console"
```

Comment:
i dont' see why the deployment guide is relevant to this docker file. i expect nobody other than possibly us will use this. i'm also not sure i agree with embedding the zac into the controller image still. it still feels wrong to me that the ziti controller docker file is not equivalent to running the `ziti` cli (because it'll come WITH the zac)... i'm still not sure this is a good idea overall